src/Pages/Warehouse/WarehousePage.js

```diff
@@ -1,21 +1,22 @@
 import { Switch, Route } from "react-router-dom";
-import Footer from "../../Components/Footer/Footer";
+import EditWarehouse from "../../Components/EditWarehouse/EditWarehouse.jsx";
 import PageHeader from "../../Components/Header/Header";
 import WarehouseList from "../../Components/WarehouseList/WarehouseList";
 import './WarehousePage.scss'
 
 function WarehousePage()  {
 
 
-    return (<>
+    return (
+    <>
         <PageHeader isActive='Warehouse' />
         <div className="component-wrapper">
                 <Switch>
                     <Route path='/' exact component={WarehouseList} />
                     <Route path='/warehouse' exact component={WarehouseList} />
                     <Route path='/warehouse/add'  component='' />
+                    <Route path='/warehouse/edit/:id' component={EditWarehouse} />
```

Comment:
Believe it or not, this has to be done or else the route doesn't work.

Reply:
I think I changed this on the inventory page because I had the same problem, forgot to do it here.